floss/main.py

```diff
@@ -198,8 +198,8 @@ def make_parser(argv):
     advanced_group.add_argument(
         "--language",
         type=str,
-        choices=[l.value for l in Language if l != Language.UNKNOWN] + ["none"],
-        default="",
+        choices=[lang.value for lang in Language],
+        default="none",
```

Comment:
does this include the unknown language in the usage output and allow to use "none"?

Reply:
Yes, this change includes the unknown. I've updated it to allow "none" and not "Unknown" in latest commit 😄